composer.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "multi-safepay/presta-shop",
+  "description": "Prestashop 1.7 compatible MultiSafepay module\n",
```

The description I want to follow the description of our Magento 2 composer file
```suggestion
  "description": "MultiSafepay module for PrestaShop 1.7",
```